makefile.shade

```diff
@@ -2,6 +2,7 @@
 var VERSION='0.1'
 var FULL_VERSION='0.1'
 var AUTHORS='Microsoft Open Technologies, Inc.'
+-BuildQuality = "";
```

Comment:
I see. Thanks for the explanation. I see this being added to KoreBuild but did not really look to closely to understand what it does...